### ISSUE TITLE
Feature Request: Incorporate CloudWatch logs

### ISSUE THREAD

**Issue by CoreyKaylor** [JetBrains/teamcity-aws-codedeploy-plugin#8]
When CodeDeploy targets are wired to log their events to CloudWatch it would be nice to bring in the logs correlated by the deployment id into teamcity's build log.

Deployment 'd-UKUR...' created

The d-UKUR will correlate with the cloudwatch log entries.

**Comment by vbedrosova** [comment]
We will probably add an option for this. But it doesn't look like the desired default behaviour because we normally try to avoid data duplication.

**Comment by CoreyKaylor** [comment]
Yeah I understand the concern. In this case, the value of having it contextual I think outweighs the drawbacks IMO.
